lib/clusterable/holding.rb

```diff
@@ -36,19 +37,43 @@ def self.holding_to_record(holding_line)
       # OCN  BIB  MEMBER_ID  STATUS  CONDITION  DATE  ENUM_CHRON  TYPE  \
       # ISSN  N_ENUM  N_CHRON  GOV_DOC UUID
       fields = holding_line.split("\t")
-      {ocn: fields[0].to_i,
-       local_id: fields[1],
-       organization: fields[2],
-       status: fields[3],
-       condition: fields[4],
-       date_received: DateTime.parse(fields[5]),
-       enum_chron: fields[6],
-       mono_multi_serial: fields[7],
-       issn: fields[8],
-       n_enum: fields[9],
-       n_chron: fields[10],
-       gov_doc_flag: !fields[11].to_i.zero?,
-       uuid: fields[12]}
+
+      if fields[3].empty?
```

Comment:
This is a long method. Since there is already some inline translation of field values going on in the Hash initializer (e.g., `DateTime.parse`) one could do class methods `self.translate_tsv_status`, `self.translate_tsv_type` (or `*_mono_multi_serial` or whatever) and `self.translate_tsv_uuid` (even though we expect it to always be `nil` I guess). That would make these procedures individually testable, or just make them private to reduce entrypoint bloat.

That would make the Hash initializer dense, but would keep semantically related items close to each other, e.g.,
```
...
mono_multi_serial: translate_tsv_mono_multi_serial(fields[7]),
...
```

I suspect this is largely a matter of taste. (I am Mr. Dense when left to my own devices)